src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -235,6 +258,8 @@ public function isSendingToQueue()
 }
 
 /**
+ * @deprecated Will be removed without replacement.
+ *
  * @param bool \$_isSendingToQueue
```

Comment:
Can you please deprecate the `setIsSendingToQueue()` as well?

Reply:
It's deprecated already https://github.com/spryker/synchronization-behavior/blob/7ce99bd9f036e64a9aa0b1d8a51f42570e4d1168/src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php#L261
But `::isSendingToQueue()` is not.